Sources/SwiftGen/Commander/XCFileListCommands.swift

```diff
@@ -0,0 +1,106 @@
+//
+// SwiftGen
+// Copyright © 2020 SwiftGen
+// MIT Licence
+//
+
+import Commander
+import PathKit
+import StencilSwiftKit
+import SwiftGenKit
+
+enum XCFileListCLI {
+  private enum CLIOption {
+    static let config = Option<Path>(
+      "config",
+      default: "swiftgen.yml",
+      flag: "c",
+      description: "Path to the configuration file to use",
+      validator: checkPath(type: "config file") { $0.isFile }
+    )
+
+    static let inputs = Option<Path?>(
+      "inputs",
+      default: nil,
+      flag: "i",
+      description: "Path to write the xcfilelist for the input files"
+    )
+
+    static let outputs = Option<Path?>(
+      "outputs",
+      default: nil,
+      flag: "o",
+      description: "Path to write the xcfilelist for the output files"
+    )
+  }
+
+  static let generate = command(
+    CLIOption.config,
+    CLIOption.inputs,
+    CLIOption.outputs
+  ) { configPath, inputPath, outputPath in
+    try ErrorPrettifier.execute {
+      let config = try Config(file: configPath)
+
+      if let inputPath = inputPath {
+        try writeInputsXCFileList(for: config, to: inputPath)
+      }
+
+      if let outputPath = outputPath {
+        try writeOutputsXCFileList(for: config, to: outputPath)
+      }
+
+      if inputPath == nil, outputPath == nil {
+        logMessage(.error, "An input or output path must be provided.")
```

Comment:
```suggestion
        logMessage(.error, "You must provide the path of an input or output xcfilelist (or both).")
```
Maybe something like this? Just to avoid confusion because "input" and "output" can mean a lot of things.